hooks/useSendMessage.ts

```diff
@@ -0,0 +1,13 @@
+import { Conversation } from '@xmtp/xmtp-js'
+
+const useSendMessage = (selectedConversation?: Conversation) => {
+  const sendMessage = async (message: string) => {

```

Comment:
Given that we don't know whether this function is going to be passed in to a memoized component or not by the caller, we should memoize it with `useCallback`